src/translations/ru.json

```diff
@@ -70,9 +73,29 @@
   "date.after": "после {from}",
   "date.before": "до {to}",
   "error.error": "Ошибка",
+  "age.exact": "{age, plural, =0 {Менее 1 года} one {{qualifier} 1 год} many {{qualifier} # лет} other {{qualifier} # года}}",
+  "age.less": "Менее {age, plural, =0 {1 года} one {1 год} many {# лет} other {# года}}",
+  "age.more": "Более {age, plural, =0 {0 лет} one {1 год} many {# лет} other {# года}}",
+  "age.between": "Между {ageFrom} {ageTo, plural, =0 {0 лет} one {1 год} many {# год} other {# лет}}",
   "error.failed_pdf": "Не удалось создать PDF-файл. Повторите попытку с меньшей диаграммой или загрузите файл SVG.",
   "error.failed_png": "Не удалось создать файл PNG. Повторите попытку с меньшей схемой или загрузите файл SVG.",
   "error.failed_to_load_file": "Ошибка загрузки файла",
   "error.failed_wikitree_load_more": "Ошибка при получении данных из WikiTree. {error}}",
+  "error.GEDCOM_READ_FAILED": "Ошибка чтения файла GEDCOM",
+  "error.ERROR_LOADING_UPLOADED_FILE": "Ошибка чтения данных. Повторите.",
+  "error.WIKITREE_ID_NOT_PROVIDED": "Не указан идентификатор WikiTree",
+  "error.WIKITREE_PROFILE_NOT_ACCESSIBLE": "Профиль WikiTree {id} недоступен",
+  "error.WIKITREE_PROFILE_NOT_FOUND": "Профиль WikiTree {id} не найден",
+  "error.failed_to_load_image": "Ошибка при открытии картинки",
   "wikitree.private": "Частный"
```

Comment:
There needs to be a comma on the end of line 90

```
"wikitree.private": "Частный",
```

Reply:
Thank you.
I use GetHub Editor.
This Editor add last empty row every time...